codegen/bin/newcoin

```diff
@@ -54,18 +54,21 @@ end
 
 def insert_blockchain_type(coin)
     target_file = "include/TrustWalletCore/TWBlockchain.h"
+    comment = " // TODO remove if the blockchain already exists, or just remove this comment if not"
```

Comment:
This line can be a constant to reuse between `insert_blockchain_type` and `insert_coin_entry`